parm/land/obs/lists/gdas_land_prototype.yaml

```diff
@@ -0,0 +1,2 @@
+observers:
+- $<< $(OBS_YAML_DIR)/adpsfc_snow.yaml
```

Comment:
Just confirming, this is the only ob that will be used (at least for initial prototyping)?

Reply:
Added the second one (ims snow).